Framework/Geometry/inc/MantidGeometry/Instrument/DetectorInfo.h

```diff
@@ -1,17 +1,16 @@
 #ifndef MANTID_GEOMETRY_DETECTORINFO_H_
 #define MANTID_GEOMETRY_DETECTORINFO_H_
 
-#include "MantidGeometry/DllConfig.h"
-#include "MantidKernel/DateAndTime.h"
-#include "MantidKernel/Quat.h"
-#include "MantidKernel/V3D.h"
-
 #include <boost/shared_ptr.hpp>
-
 #include <mutex>
 #include <unordered_map>
 #include <vector>
 
+#include "MantidGeometry/DllConfig.h"
+#include "MantidKernel/DateAndTime.h"
+#include "MantidKernel/Quat.h"
+#include "MantidKernel/V3D.h"
```

Comment:
Can you please ensure that all of the includes are really needed.

Reply:
I think those includes were already part of the header file - I think clang format rearranged them.

Reply:
Ok. still review if needed.